crypto-ffi/bindings/js/wdio.conf.ts

```diff
@@ -1,14 +1,18 @@
+import { tmpdir } from "os";
 import fs from "fs/promises";
 import path from "path";
 import { fileURLToPath } from "url";
 import logger from "@wdio/logger";
 
-const STATIC_SEVER_URL = "http://localhost:3000/";
-
-const staticPath = path.resolve(process.cwd(), `../platforms/web`);
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const STATIC_SERVER_URL = "http://localhost:3000/";
 const log = logger("wdio.conf.ts");
 
+let staticPath;
+if (staticPath === undefined) {
```

Comment:
This looks like an unconditional branch?

Reply:
Thanks for asking about that. That was supposed to address the issue where `wdio.conf.ts` is imported by the main wdio process, as well as worker processes, which would then needlessly create multiple temporary directories.

However, as written, that didn't really work so I changed it a bit so we only really create one tempdir per wdio run. I also added a comment explaining what's going on.